etc/kayobe/kolla/config/prometheus/smart.rules

```diff
@@ -1,6 +1,10 @@
 {% raw %}
 
-- alert: DiskSmartStatusUnhealthy
+groups:
+- name: Smart
```

Comment:
This looks fine when comparing to an example from https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/:

```
groups:
- name: example
  rules:

  # Alert for any instance that is unreachable for >5 minutes.
  - alert: InstanceDown
    expr: up == 0
    for: 5m
    labels:
      severity: page
    annotations:
      summary: "Instance {{ $labels.instance }} down"
      description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes."
```